app/controllers/api/v1/games_controller.rb

```diff
@@ -9,9 +9,13 @@ def show
   end
 
   def create
-    game = Game.create(game_params)
-    User.assign_game(current_user.id, game.id)
-    render json: game
+    @game = Game.new(game_params)
+    if @game.save
+      User.assign_game(current_user.id, @game.id)
+      render json: @game
+    else
+      render json: { errors: @game.errors }, status: 500
```

Comment:
I think the status should be a 400-level, not 500. probably `:unprocessable_entity`